controllers/gce/instances/instances.go

```diff
@@ -99,23 +102,27 @@ func (i *Instances) AddInstanceGroup(name string, port int64) ([]*compute.Instan
 			glog.V(3).Infof("Instance group %v already exists in zone %v", name, zone)
 		}
 
-		found := false
+		existingPorts := map[int64]bool{}
 		for _, np := range ig.NamedPorts {
-			if np.Port == port {
+			existingPorts[np.Port] = true
+		}
+		newPorts := []*compute.NamedPort{}
+		for _, np := range namedPorts {
+			if existingPorts[np.Port] {
 				glog.V(3).Infof("Instance group %v already has named port %+v", ig.Name, np)
-				found = true
-				break
+				continue
```

Comment:
`glog.V(3).Infof("Instance group %v already has named port %+v", ig.Name, np)` occurs very frequently. Recommend we bump the level to 5. Thoughts?

Reply:
Sure. You look at logs a lot more than me, so happy to do whatever you say :)
Updated to 5.